java/ql/src/semmle/code/java/NumberFormatException.qll

```diff
@@ -0,0 +1,71 @@
+import java
```

Comment:
```suggestion
/** Provides classes and predicates for reasoning about `java.lang.NumberFormatException`. */
import java
```

Reply:
Thanks @aschackmull. I've updated the qldoc.